app/presenters/socure_error_presenter.rb

```diff
@@ -150,20 +155,28 @@ def heading_string_for(error_code)
   end
 
   def error_string_for(error_code)
-    if error_code == :network
-      return t('doc_auth.errors.general.new_network_error')
-    end
-
-    remapped_error_code = remapped_error(error_code)
-    if remapped_error_code == 'underage' # special handling because it says 'Login.gov'
-      I18n.t('doc_auth.errors.underage', app_name: APP_NAME)
+    case error_code
+    when :network
+      t('doc_auth.errors.general.new_network_error')
+    when :timeout
+      t('idv.errors.try_again_later')
     else
-      # i18n-tasks-use t('doc_auth.errors.unreadable_id')
-      # i18n-tasks-use t('doc_auth.errors.unaccepted_id_type')
-      # i18n-tasks-use t('doc_auth.errors.expired_id')
-      # i18n-tasks-use t('doc_auth.errors.low_resolution')
-      # i18n-tasks-use t('doc_auth.errors.id_not_found')
-      I18n.t("doc_auth.errors.#{remapped_error_code}")
+      if remapped_error(error_code) == 'underage' # special handling because it says 'Login.gov'
+        I18n.t('doc_auth.errors.underage', app_name: APP_NAME)
+      else
+        # i18n-tasks-use t('doc_auth.errors.unreadable_id')
+        # i18n-tasks-use t('doc_auth.errors.unaccepted_id_type')
+        # i18n-tasks-use t('doc_auth.errors.expired_id')
+        # i18n-tasks-use t('doc_auth.errors.low_resolution')
+        # i18n-tasks-use t('doc_auth.errors.id_not_found')
+        I18n.t("doc_auth.errors.#{remapped_error(error_code)}")
+      end
     end
   end
+
+  def in_person_enabled?
+    IdentityConfig.store.in_person_doc_auth_button_enabled &&
+      Idv::InPersonConfig.enabled_for_issuer?(issuer) &&
+      flow_path == :standard
```

Comment:
```suggestion
      flow_path == 'standard'
```
looks like flow_path is a string ... are we not offering IPP in the hybrid flow?

Reply:
So Jmax and I just finished pairing on this:

1. The hybrid path can't use the errors_controller since it touches the idv_session which fails on hybrid. We reintroduced the `errors` action on the Idv::HybridMobile::Socure::DocumentCaptureController.
2. Seems like the hybrid flow should allow access to the Socure doc capture, then return to the desktop and allow any errors to be shown there. Then `Try again` could re-enter the hybrid flow.

Meet Monday to discuss?

Reply:
1. make sense regarding the idv_session 👍🏿